goodreads-predictor/src/goodreads_predictor/pipelines/feature_engineering/nodes.py

```diff
@@ -5,6 +5,13 @@
 from typing_extensions import Tuple, List, Union, Dict, Any
 import pandas as pd
 import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
Please remove libraries not being used (matplotlib, seaborn, wordcloud, and sklearn StandardScaler)